docs/deployment/gae_flex_datastore.rst

```diff
@@ -0,0 +1,167 @@
+.. _appengine_flexible_tutorial:
+
+Google App Engine Flexible with Datastore and :app:`Pyramid`
+============================================================
+
+It is possible to run a :app:`Pyramid` application on `Google App Engine <https://cloud.google.com/appengine/>`_.  This tutorial is written "environement agnostic" meaning the commands here should work on Linux, Mac or Windows. This tutorial also assumes you've already installed and created a :app:`Pyramid` application, and that you have a Google App Engine account.
+
+Setup
+-----
+
+First we'll need to set up a few things in App Engine. If you don't need Datastore access for your project or any other GCP service, you can skip the Credentials section.
+
+Credentials
+###########
+
+Navigate to App Engine's IAM And Admin section and click on Service Accounts in the left sidebar, then create a `Service Account <https://cloud.google.com/iam/docs/service-accounts>`_.
+
+Once a service account is created, you will be given a ``.json`` key file. This will be used to allow your Pyramid application to communicate with GCP services. Move this file to your Pyramid project. A best practice here would be to make sure this file is listed in ``.gitignore`` so that it's not checked in with the rest of your code.
+
+Now that we have a service account, we'll need to give it a couple of roles. Click *IAM* in the left sidebar of *IAM And Admin*. Find the service account you've just created and click the Edit button. Give this account the *Cloud Datastore User* role for read/write access. For read-only access, give it *Cloud Datastore Viewer*.
```

Comment:
All GUI interface items should use `:guilabel:`, for example:

```
Click :guilabel:`IAM` in
```

For this paragraph, please do so for `IAM`, `IAM And Admin`, and `Edit`.

I am not sure whether "Cloud Datastore User" and "Cloud Datastore Viewer" are UI elements or literals. If they are literals, then they should use inline rst syntax like so:

```
``Cloud Datastore Viewer``
```

Reply:
I suppose they are both, technically? https://i.imgur.com/KFy6qnu.png

Reply:
It looks like a UI element to me, like an option in a select menu that can be clicked, so `:guilabel:` is the way to go. Inline literals are usually for stuff like "enter the command `wherecani get-a-haircut`"